lib/please_cli/please_cli/run.py

```diff
@@ -302,7 +300,7 @@ def cmd(ctx,
         )
 
         for require in project_config.get('requires', []):
-            env_name = '{}_URL'.format(please_cli.utils.normalize_name(require).upper())
+            env_name = f'{please_cli.utils.normalize_name(require).upper()}_URL'
```

Comment:
hard to read too

Reply:
Indeed, in cases like this it'd be better to have a temporary variable, or to keep format

Reply:
I have kept the old format in cases like these.